roles/backend/templates/backend.yml.j2

```diff
@@ -16,4 +16,48 @@ log-level: {{ backend_log_level }}
 {% endif %}
 {# -------------------------------------- #}
 state-dir: "/var/lib/sensu/sensu-backend"
+config-file: "/etc/sensu/backend.yml"
```

Comment:
Maybe we should group all backend-related variables into a single variable `backend_config` and loop over the contents here? Alternatively, we can expose the `backend_config` to the user and document what options are available. (I am more partial to the second solution because it is more straightforward.)

If we group the variables, the whole template reduces to:

```
{% for name, value in backend_config.items() %}
{%   if value %}
{{ name }}: {{ value }}
{%   endif %}
{% endfor %}
```

Reply:
Thank you for these suggestions. I decided to go for both, so we now mostly generate the contents of the `/etc/sensu/backend.yml` from the `backend_config` items.